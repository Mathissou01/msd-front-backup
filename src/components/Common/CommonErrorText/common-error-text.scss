@use "src/styles/01-settings/settings.colors" as colors;

.c-CommonErrorMessage {
  padding: 2px 0;
  color: colors.$error;
}
