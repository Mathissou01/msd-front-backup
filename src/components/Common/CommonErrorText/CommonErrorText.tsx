import React from "react";
import "./common-error-text.scss";

interface ICommonErrorTextProps {
  message: string;
  errorId: string;
}

export default function CommonErrorText({
  message,
  errorId,
}: ICommonErrorTextProps) {
  return (
    <em className="c-CommonErrorMessage" role="alert" id={errorId}>
      {message}
    </em>
  );
}
