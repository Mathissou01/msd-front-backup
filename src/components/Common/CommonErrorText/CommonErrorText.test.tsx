import { render } from "@testing-library/react";
import CommonErrorText from "./CommonErrorText";

const mock = {
  message: "error message",
  errorId: "error-1",
};

describe("CommonErrorText", () => {
  it("renders", () => {
    const { container } = render(
      <CommonErrorText message={mock.message} errorId={mock.errorId} />,
    );
    expect(container).toMatchSnapshot();
  });
});
