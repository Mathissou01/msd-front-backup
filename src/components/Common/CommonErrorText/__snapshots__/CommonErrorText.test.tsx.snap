// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CommonErrorText renders 1`] = `
<div>
  <em
    class="c-CommonErrorMessage"
    id="error-1"
    role="alert"
  >
    error message
  </em>
</div>
`;
